coqpp/coqpp_lex.mll

```diff
@@ -103,6 +103,7 @@ rule extend = parse
 | "DOC_GRAMMAR" { DOC_GRAMMAR }
 | "END" { END }
 | "DECLARE" { DECLARE }
+| "NOT" { NOT }
```

Comment:
Don't add a token for that, you should probably parse it reusing another one.

Reply:
(For instance, `DECLARE GLOBAL PLUGIN` doesn't look too bad to my eyes.)

Reply:
That doesn't make sense, it's saying "this is statically linked, not a plugin"
How about `DECLARE STATICALLY LINKED`?

Reply:
GLOBAL is close to static semantically.

Reply:
changed